huaweicloud/services/cdn/data_source_huaweicloud_cdn_analytics.go

```diff
@@ -0,0 +1,152 @@
+// Generated by PMS #241
+package cdn
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/tidwall/gjson"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/helper/httphelper"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/helper/schemas"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func DataSourceCdnAnalytics() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceCdnAnalyticsRead,
+
+		Schema: map[string]*schema.Schema{
+			"action": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `"Specifies the action name. Possible values are **summary** and **detail**.`,
```

Comment:
```suggestion
				Description: `Specifies the action name. Possible values are **summary** and **detail**.`,
```
please check the other places.

Reply:
OK